WordPress/src/main/java/org/wordpress/android/ui/posts/PostSettingsInputDialogFragment.java

```diff
@@ -94,34 +93,32 @@ public void onDismiss(DialogInterface dialog) {
     public Dialog onCreateDialog(Bundle savedInstanceState) {
         AlertDialog.Builder builder =
                 new MaterialAlertDialogBuilder(new ContextThemeWrapper(getActivity(), R.style.PostSettingsTheme));
-        LayoutInflater layoutInflater = getActivity().getLayoutInflater();
+        LayoutInflater layoutInflater = requireActivity().getLayoutInflater();
```

Comment:
Minor (🔍): Although this change is perfectly valid, consider avoid adding an unrelated change on a single commit. If you need to make this change, add a separate targeted commit on top of the main set of commits on a PR. Then, explaining the reasoning behind that change, even maybe group other such changes to make this change more impactful. How does this sound? 🙏

PS: No need for you to do anything here, this is just for educational purposes only.